inference-engine/cmake/dependencies.cmake

```diff
@@ -263,6 +276,16 @@ if (ENABLE_OPENCV)
         log_rpath_from_dir(OPENCV "${OpenCV_DIR}/../lib")
     endif()
     debug_message(STATUS "opencv=" ${OPENCV})
+
+    install(DIRECTORY ${OPENCV}/
+            DESTINATION opencv
+            COMPONENT opencv
+            PATTERN ie_dependency.info EXCLUDE)
+            
+    install(DIRECTORY ${OPENCV}/../python/
+            DESTINATION python
+            COMPONENT opencv)
```

Comment:
@asuhov @alalek I think we should not install this version of OpenCV. It's used only for tests..

Reply:
Agree, this version is usually a bit outdated. But it's convenient to have it in the package to run samples.

Reply:
We should install ALL components that a part of package now, and resolve other issues step by step...
1st priority - no actions out of CMake for preparing FULL package.